inc/compat/plugins/compat-plugin-woo-delivery.php

```diff
@@ -43,8 +63,12 @@ public function late_hooks() {
 		$this->remove_action_for_class( 'woocommerce_review_order_before_payment', array( 'Coderockz_Woo_Delivery_Public', 'coderockz_woo_delivery_add_custom_field' ), 10 );
 		$this->remove_action_for_class( 'woocommerce_checkout_before_order_review_heading', array( 'Coderockz_Woo_Delivery_Public', 'coderockz_woo_delivery_add_custom_field' ), 10 );
 
-		// Get delivery date value from session
-		add_filter( 'woocommerce_checkout_get_value', array( $this, 'change_default_field_values_from_session' ), 200, 2 );
+		// TODO: CHECK WHY IT WAS REMOVED
+		// // Get delivery date value from session
+		// add_filter( 'woocommerce_checkout_get_value', array( $this, 'change_default_field_values_from_session' ), 200, 2 );
```

Comment:
@vlad-dedikov, why was this removed?

Reply:
@vlad-dedikov, maybe the function `maybe_set_datepicker_fields_session_values` is replacing what this code should be doing.

Check the behavior of the plugin and if it is working well we can remove this code entirely.

Reply:
@vlad-dedikov, I restored this part of the code since it does not seem to have any negative effects.